CreeDictionary/tests/API_tests/model_test.py

```diff
@@ -200,3 +202,25 @@ def test_filter_cw_content():
     assert (
         len(list(filtered)) == 0
     )  # nipa should no longer be there because the preverb nipa is a MD only word
+
+
+@pytest.mark.django_db
+def test_paradigm():
+    def deep_contain(container: Iterable, testee):
+        """check if `testee` is in any depth of `container`"""
```

Comment:
I only had `RecersionError: maximum recursion depth exceeded` once when I write this function